formatters/swiftcov/Calculator.swift.gcov

```diff
@@ -0,0 +1,64 @@
+    -:    0:Source:/Users/katsumi/Dropbox/Xcode Projects/SwiftCov/Examples/ExampleFramework/ExampleFramework/Calculator.swift
```

Comment:
Do the other formatters include example files right alongside source files? I would've expected this to live in `examples/`